File: Sorting/Selection.py
def SelectionSort(arr):
    for i in range(len(arr)-1):
        min = i
        for j in range(i+1, len(arr)):
            if arr[min] > arr[j]:
                min = j
        if min != i:
            arr[min], arr[i] = arr[i], arr[min]

arr = [7,3,9,1,4,2]
print(arr)
SelectionSort(arr)
print(arr)

File: Sorting/Selection.cpp
#include<iostream>
#include<vector>
using namespace std;

void SelectionSort(vector<int> &arr){
    for(int i = 0; i<arr.size()-1; i++){
        int min = i;
        for(int j = i + 1; j<arr.size(); j++){
            if(arr[j] < arr[min]){
                min = j;
            }
        }
        if(min != i){
            int t = arr[min];
            arr[min] = arr[i];
            arr[i] = t;
        }
    }
}

int main(){
    vector<int> arr = {3,2,6,1,5};
    SelectionSort(arr);
    for(int ele : arr){
        cout<<ele<<", ";
    }
}